zerver/import_data/slack/slack.py

```diff
@@ -19,8 +19,8 @@
 from typing import Any, Dict, List, Optional, Tuple
 from zerver.forms import check_subdomain_available
 from zerver.models import Reaction, RealmEmoji, Realm, UserProfile
-from zerver.lib.slack_message_conversion import convert_to_zulip_markdown, \
-    get_user_full_name
+from zerver.import_data.slack.slack_message_conversion import \
+    convert_to_zulip_markdown, get_user_full_name
```

Comment:
I think it'd be better to just call this `zerver/data_import/slack.py`; I'm not sure there's really a need for a separate directory.  Or if you do a directory, the convention would be to call it `zerver/data_import/slack/__init__.py`, but given that most of these import tools are going to be 1-2 files, I think it's cleaner to just have e.g. `zerver/data_import/slack.py` as the main file, and the format conversion be `zerver/data_import/slack_message_conversion.py`.